sylt-parser/src/statement.rs

```diff
@@ -211,28 +211,37 @@ pub fn path<'t>(ctx: Context<'t>) -> ParseResult<'t, Identifier> {
     Ok((ctx, Identifier { span, name: result }))
 }
 
-pub fn use_path<'t>(ctx: Context<'t>) -> ParseResult<'t, (Identifier, PathBuf)> {
+pub fn use_path<'t>(ctx: Context<'t>) -> ParseResult<'t, (Identifier, FileOrLib)> {
     let (ctx, path_ident) = path(ctx)?;
     let path = &path_ident.name;
     let name = path
         .trim_start_matches("/")
         .trim_end_matches("/")
         .to_string();
     let file = {
-        let parent = if path.starts_with("/") {
-            ctx.root
+        if let Some(name) = library_name(&name) {
```

Comment:
Does this prioritize libraries? I think the files should be prioritized whenever possible

Reply:
This can be changed, I'd prefer a warning